lib/mapRoutes.js

```diff
@@ -177,3 +194,21 @@ module.exports = function (app) {
 
   return app
 }
+
+function validatePrefix (urlPrefix) {
+  // remove whitespace
+  if (typeof urlPrefix === 'string') {
+    urlPrefix = urlPrefix.replace(/\s/g, '')
```

Comment:
why not use `trim()`?

Reply:
`.trim()` only removes whitespace from both ends of the string, `.replace()` will allow us to remove the whitespace from the middle of the string as well